src/features/auth/WalletAuthModal.tsx

```diff
@@ -121,19 +124,29 @@ export const WalletAuthModal = ({ open }: { open: boolean }) => {
   };
 
   useEffect(() => {
-    if (
-      // safe to refer to window here because we are in useEffect -g
-      window.location.search === AUTO_OPEN_WALLET_DIALOG_PARAMS ||
-      walletAuth.connectorName
-    ) {
-      walletAuth.connectorName && activate(walletAuth.connectorName);
-    }
+    if (walletAuth.connectorName && !web3Context.active)
+      activate(walletAuth.connectorName);
   }, []);
 
+  const inject = async () => {
```

Comment:
I guess I'd prefer to leave this out of `main` when merging this PR, and add it later if there isnt any functionaltiy here yet...

Reply:
i understand. it's just here in this case because i was going back and forth between this and #1704 a lot and it felt right to keep some placeholder here for when i start working on #1705 more